projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.scss

```diff
@@ -0,0 +1,44 @@
+.jetpack-email-preview {
+  height: 200px;
+  width: 645px;
+  left: 391px;
+  top: 392px;
+  border-radius: 0;
```

Comment:
Let's stick to core's version, and share screenshot with @crisbusquets to see if the core version differs from design. Just to avoid component overrides.

Reply:
Update: we stuck to core version and in [core we're changing to 4px rounding](https://github.com/WordPress/gutenberg/pull/51254).